packages/cli/src/commands/deploy.ts

```diff
@@ -261,3 +281,43 @@ const isBuildable = (path: string) => {
 
   return true
 }
+
+const selectApiScope = async (token: string): Promise<Scope> => {
+  const accessibleToMySpace = await StoryblokClient({
+    token,
+    scope: 'my-space',
+  }).isAuthenticated()
+
+  const accessibleToPartnerPortal = await StoryblokClient({
+    token,
+    scope: 'partner-portal',
+  }).isAuthenticated()
+
+  if (!accessibleToMySpace && !accessibleToPartnerPortal) {
+    console.error(
+      red('[ERROR]'),
+      `The given token(\`${token}\`) seems to be invalid.`,
```

Comment:
```suggestion
      `The token appears to be invalid as it does not have access to either My Plugins or the plugins on the Partner Portal.`,
```

A suggestion is to state the reason why we think it's invalid, and to not print the token in the console, as it is a secret.